packages/core/src/Stores/client-store.js

```diff
@@ -5,6 +5,7 @@ import ObjectUtils from '@deriv/shared/utils/object';
 import { getUrlSmartTrader } from '@deriv/shared/utils/storage';
 import { requestLogout, WS } from 'Services';
 import ClientBase from '_common/base/client_base';
+import * as Login from '_common/base/login';
```

Comment:
[optional] I can only find an instance of `redirectToLogin` method being used from this import, do we need to actually import all exported functions?

Reply:
Nice catch sir. Yeah, we actually don't need that.